svm/src/transaction_execution_result.rs

```diff
@@ -51,7 +50,7 @@ impl ExecutedTransaction {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Eq, PartialEq)]
 pub struct TransactionExecutionDetails {
```

Comment:
It would be nice if we could remove this struct entirely, now that we have `struct CommittedTransaction` which is almost identical. 

I think we've got too many struct types :)

Reply:
Sorry I'm a struct maxi